tests/dragonfly/connection_test.py

```diff
@@ -1160,3 +1160,54 @@ async def push_pipeline(bad_actor_client, size=1):
             info = await good_client.info()
 
     assert info["dispatch_queue_bytes"] == 0
+
+
+async def test_client_unpause(df_factory):
+    server = df_factory.create()
+    server.start()
+
+    async_client = server.client()
+    await async_client.client_pause(3000, all=False)
+
+    async def set_foo():
+        client = server.client()
+        async with async_timeout.timeout(2):
+            await client.execute_command("SET", "foo", "bar")
+
+    p1 = asyncio.create_task(set_foo())
+
+    await asyncio.sleep(0.5)
+    assert not p1.done()
+
+    async with async_timeout.timeout(0.5):
+        await async_client.client_unpause()
+
+    await p1
```

Comment:
lets add a timeout here that this await returns in 0.5 second

Reply:
+1